javaparser-symbol-solver-testing/src/test/resources/javaparser_methodcalls_expected_output/com_github_javaparser_ParseProblemException.txt

```diff
@@ -1,10 +1 @@
   Line 21) createMessage(assertNotNull(problems)) ==> com.github.javaparser.ParseProblemException.createMessage(java.util.List<com.github.javaparser.Problem>)
-  Line 21) assertNotNull(problems) ==> com.github.javaparser.utils.Utils.assertNotNull(T)
```

Comment:
Same here. Well, I suspect the other hundreds of files are the generated ones so I will not look further

Reply:
What does it mean to have more or less results here? The tests take ages, and for some reason they all work fine from the IDE. It's hard to debug.

Reply:
If from the IDE the results are different could it be that it uses different paths or a different parallelism I suppose. Definitely very weird.